docs/api/api/jobstatus.rng

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grammar
+    xmlns="http://relaxng.org/ns/structure/1.0"
+    xmlns:a="http://www.example.com/annotation"
+    ns="http://buildservice.org/api"
+    datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
+
+  <include href="obs.rng"/>
+
+  <start>
+    <ref name="jobstatus-element"/>
+  </start>
+
+  <define ns="" name="jobstatus-element">
+    <element ns="" name="jobstatus">
+      <a:documentation>
+        This element stores the information of the currently active job.
+
+        All attributes and elements are only present if a job is currently
+        running, otherwise the reply is empty.
+      </a:documentation>
+
+      <optional>
+        <attribute name="code">
+          <ref name="repository-code"/>
+        </attribute>
+      </optional>
+
+      <optional>
+        <element name="starttime">
+          <a:documentation>
+            The unix time at which the build started.
+          </a:documentation>
+          <data type="nonNegativeInteger"/>
+        </element>
+      </optional>
+
+      <optional>
+        <element name="lastduration">
+          <a:documentation>
+            Time in seconds that the last build took.
+          </a:documentation>
+          <data type="nonNegativeInteger"/>
+        </element>
+      </optional>
+
+      <optional>
+        <element name="lastduration">
```

Comment:
Isn't this optional element defined from line 47 to 54 repeated from line 38 to 45?

Reply:
In fact every attribute and element of `jobstatus` is optional: if no job is running at the moment, then you get just `<jobstatus/>` back from obs. I have added them around every optional element & attribute, as I was unsure if you can wrap them all in one optional.

Reply:
Ups, I misread your comment and the `lastduration` element was duplicated. Should be fixed now.